src/accelerate/utils/operations.py

```diff
@@ -219,6 +222,12 @@ def gather(tensor):
     Returns:
         The same data structure as `tensor` with all tensors sent to the proper device.
     """
+    if not is_torch_distributed_available():
+        raise ModuleNotFoundError(
+            "`torch` was compiled without distributed support (`torch.distributed.is_available() == False`). "
+            "Please install a version which includes distributed support in order to call gather."
+        )
```

Comment:
Why do we need to raise here? Users can definitely call `gather` on a single process with distributed support. That's tje promise of Accelerate: the script will run **the same and without errors** on any setup when you use accelerate primitives.

Since we can't be launched distributed-ly without distributed support, we should just leave this as is and the last elese with return the `tensor` or am I missing something?

Reply:
Great point, yes I believe we should just return the tensor and `noop`

Reply:
Didnt have this philosophy in mind, will do the changes 👌